packages/wdio-sauce-service/src/launcher.js

```diff
@@ -51,32 +51,36 @@ export default class SauceLauncher {
             }
         }
 
-        if (Array.isArray(capabilities)) {
-            for (const capability of capabilities) {
-                if (!capability['sauce:options']) {
-                    capability['sauce:options'] = {}
-                }
+        function prepareCapability(capability) {
```

Comment:
Can we move this function into `utils.js`?

Reply:
Sorry, just so I understand, are you suggesting putting this function in a new file `utils.js` in `wdio-sauce-service`, or putting this function in `packages/webdriverio/src/utils.js`?

Reply:
As this is specific to the Sauce service I suggest to the `utils.js` in the `wdio-sauce-service` package. I know that we modify capabilities in other services too but refactoring that should be part of a different PR.

Reply:
I've made this change into a separate commit https://github.com/webdriverio/webdriverio/pull/5610/commits/bc4762f99d419d794894b52874cd90e5fb28b53c